yarpl/test/Observable_test.cpp

```diff
@@ -250,30 +252,32 @@ TEST(Observable, CancelFromDifferentThread) {
   LOG(INFO) << "cancelled after " << emitted << " items";
 }
 
-TEST(Observable, toFlowable) {
-  auto a = Observable<int>::create([](Reference<Observer<int>> obs) {
-    obs->onNext(1);
-    obs->onComplete();
-  });
-
+TEST(Observable, toFlowableDrop) {
+  auto a = Observables::range(1,10);
   auto f = a->toFlowable(BackpressureStrategy::DROP);
 
-  std::vector<int> v;
-  f->subscribe(yarpl::flowable::Subscribers::create<int>(
-      [&v](const int& value) { v.push_back(value); }));
+  std::vector<int64_t> v;
 
-  EXPECT_EQ(v.at(0), 1);
+  auto subscriber = make_ref<testing::StrictMock<yarpl::flowable::MockSubscriber<int64_t>>>(5);
+
+  EXPECT_CALL(*subscriber, onSubscribe_(_));
```

Comment:
```cpp
{
  ::testing::InSequence ensure_order;
  EXPECT_CALL(...);
  EXPECT_CALL(...);
}
``` 
to verify that these calls take place in the order you `EXPECT_CALL` them in (unless StrictMock does this already? looks like it only treats unEXPECT'd calls as errors).

Reply:
yeah, I know about InSequence. I would argue that in this test, I am not testing the sequence of calls. That is tested somewhere else.

Reply:
Fair enough